cartridges/int_starwars_api/cartridge/scripts/services/deathStarData.js

```diff
@@ -0,0 +1,23 @@
+function getDeathStarData() {
+    var LocalServiceRegistry = require('dw/svc/LocalServiceRegistry');
+
+    var deathStarData = LocalServiceRegistry.createService("http.starwars.deathstardata", {
+        createRequest: function (svc, args) {
+            svc.setRequestMethod("GET");
+            return args
+        },
+        parseResponse: function (svc, client) {
+            return client.text
+        },
+        filterLogMessage(msg) {
```

Comment:
This doesn't look like the right way to declare a callback, does it work?

Reply:
fixed, it but forgot to push it, it is pushed now